examples/platforms/da15000/cxx-symbols.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
We don't have startup-gcc.c. file.  It could be possible to align with cc2538 but it would be removing one file and creating another one.